CHANGELOG.md

```diff
@@ -1,3 +1,93 @@
+# v0.10.2
+
+## Announcing Akri v0.10.2!
+Akri v0.10.2 is a pre-release of Akri.
+
+To find out more about Akri, check out our [documentation](https://docs.akri.sh/) and start
+[contributing](https://docs.akri.sh/community/contributing) today!
+
+## New Features
+The v0.10.2 release contains the following changes:
+
+1. **Enable mounting connectivity information for multiple devices/instances in a Pod** (https://github.com/project-akri/akri/pull/560 , https://github.com/project-akri/akri/pull/561). Previously, Akri could only mount one device property per protocol to a Pod as all devices of the same protocol had the same environment variable name. This release fixes this issue by appending the instance hash to the environment variable name and slot ID to the annotation key name. This is a **breaking change** as it changes the way brokers look up properties.
+2. **Enable udev discovery handler to discover multiple node devices** (https://github.com/project-akri/akri/pull/564). Akri now allows udev discovery handler to group devices that share a parent/child relation.
+3. **Mount udev devpath in Akri brokers** (https://github.com/project-akri/akri/pull/534). This enables discovering udev devices without a devnode by using devpath instead. This is a **breaking change** in the udev discovery handler as it changes the way Akri creates instance ids for udev devices.
+
+**Fixes, features, and optimizations**
+- fix: OPC discovery handler uses discoveryURL specified by users if the return applicationURL from OPC server is not resolvable (https://github.com/project-akri/akri/pull/570)
+- fix: Fixed watch crash API unreachable (https://github.com/project-akri/akri/pull/568)
+- fix: Added udev devnode to device mounts instead of devpath (https://github.com/project-akri/akri/pull/562)
+- fix: Modify Agent to reduce frequency of Pods getting UnexpectedAdmissionError (https://github.com/project-akri/akri/pull/556)
+- opt: Use tokio::sync::RwLock instead of tokio::sync::Mutex (https://github.com/project-akri/akri/pull/541)
+- opt: ListAndWatch only sends device if the list has changed (https://github.com/project-akri/akri/pull/540)
+- opt: Added more securityContext to ensure Helm templates use the most restrictive setting (https://github.com/project-akri/akri/pull/547)
+- feat: Add nodeSelectors for Akri agent (https://github.com/project-akri/akri/pull/536)
+
```

Comment:
I believe we have a few more that have recently come in. I'd look at the latest merged PRs or this changelog: https://github.com/project-akri/akri/compare/v0.8.23...main

Reply:
updated to add more fixes. also sorting this list from old to new